src/control/server/storage/scm/faults.go

```diff
@@ -65,6 +65,11 @@ var (
 		"request included already-mounted mount target (cannot double-mount)",
 		"unmount the target and retry the operation",
 	)
+	FaultFormatMissingDevice = scmFault(
```

Comment:
What do you think about linking them to the docs that describe the SCM preparation procedure?

Reply:
It's not a bad idea, but I'd rather do that as part of a larger effort focused on supportability improvements. Right now we don't have a consistent way of surfacing the resolution strings.